accessibility/README.md

```diff
@@ -0,0 +1,19 @@
+# Accessibility
+
+## What is it?
+
+Web accessibility, [according to the W3C](https://www.w3.org/WAI/intro/accessibility.php):
+
+> means that people with disabilities can use the Web. More specifically, Web accessibility means that people with disabilities can perceive, understand, navigate, and interact with the Web, and that they can contribute to the Web. Web accessibility also benefits others, including older people with changing abilities due to aging.
+
+For organisations and website owners, being accessible means that your site can be used by a range of people with varying types and severities of disability. For the web, the most relevant types of disabilities include visual impairments (e.g. blindness, colour blindness, nearsightedness), cognitive impairments (e.g. dementia), motor impairments (particularly those affecting the hands and arms), learning difficulties (e.g. dyslexia), and hearing impairments. An accessible site should also work well with [assistive technology](https://www.atia.org/at-resources/what-is-at/) such as screen readers, dictation software, magnification software, braille terminals and alternative input devices. An accessibile site may comform to accessibility standards such as W3C's [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG20/) (WCAG). However an accessible site is not necessarily WCAG-compliant, and a WCAG-compliant site is not necessarily accessible.
```

Comment:
I like the intention but 'highly-dynamic' and 'progressive enhancement' both will need further explanation for many.